packages/components/src/form-toggle/README.md

```diff
@@ -1,22 +1,56 @@
 # FormToggle
+FormToggle switches a single setting on or off.
 
-The Form Toggle Control is a switch that should be **used when the effect is boolean and instant**. The Form Toggle Control is a complement to the Checkbox Control.
+![On and off FormToggles](https://wordpress.org/gutenberg/files/2019/01/Toggle.jpg)
 
-Use Form Toggle when:
+## Table of contents
 
-- The effect of the switch is immediately visible to the user. For example: when applying a drop-cap to text, the actual drop-cap is immediately activated.
-- There are only two states for the switch: on or off.
+1. [Design guidelines](http://#design-guidelines)
```

Comment:
Looks like these anchor links have an unnecessary `http://` in them.

Reply:
Good catch 👍